CMakeLists.txt

```diff
@@ -17,7 +18,12 @@ option(WARNINGS_AS_ERRORS "Treat warnings as errors" OFF)
 option(CAPI_STDCALL "Set calling convention of C API callbacks stdcall" OFF)
 option(SCTP_DEBUG "Enable SCTP debugging output to verbose log" OFF)
 
-if(USE_GNUTLS)
+if (USE_MBEDTLS AND USE_GNUTLS)
+	message(FATAL_ERROR "Both USE_MBEDTLS and USE_GNUTLS can not be enabled at the same time")
+endif()
+
+
+if(USE_GNUTLS OR USE_MBEDTLS)
```

Comment:
Same here, Nettle should not be linked against with MbedTLS.

Reply:
:+1: Fixed, removed the nettle usage for mbed tls